app/views/insights/_code_frequency.html.erb

```diff
@@ -0,0 +1,3 @@
+<!DOCTYPE html>
```

Comment:
What does the first row do? Do we need it?

Reply:
solved